xskillscore/core/deterministic.py

```diff
@@ -112,6 +114,106 @@ def pearson_r(a, b, dim, weights=None):
     )
 
 
+def spearman_r(a, b, dim, weights=None):
+    """
+    Spearman's correlation coefficient.
+
+    Parameters
+    ----------
+    a : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    b : xarray.Dataset or xarray.DataArray
+        Labeled array(s) over which to apply the function.
+    dim : str, list
+        The dimension(s) to apply the correlation along.
+    weights : xarray.Dataset or xarray.DataArray
+        Weights matching dimensions of ``dim`` to apply during the function.
+        If None, an array of ones will be applied (i.e., no weighting).
+
+    Returns
+    -------
+    xr.DataArray or xr.Dataset
+        Pearson's correlation coefficient.
```

Comment:
Spearman's

Reply:
Also change these returns to `xarray.DataArray or xarray.Dataset` for consistency with above. This is a typo from one of my docstrings that I fixed in #31 I believe.